Update docs to reflect changes in how CF templates are delivered to Customer (#3953)
gitpod-io/website: src/content/docs/gitpod-dedicated/guides/getting-started.md

You will need to execute two CloudFormation templates to install the infrastructure and subsequently Gitpod Dedicated:

> 1. `gitpod-role` Template: This template creates a new IAM role with ten policies attached. These policies grant the minimum permissions necessary to install and run Gitpod Dedicated in your account.

Author: ```suggestion
1. `gitpod-role` Template: This template creates a new IAM role with specific policies attached. These policies grant the minimum permissions necessary to install and run Gitpod Dedicated in your account.
```

> 2. `gitpod-instance` Template: This template installs the infrastructure for Gitpod Dedicated. The role created by the first template is used when installing this second template.

Reviewer: ```suggestion
2. `gitpod-instance` Template: This template installs the infrastructure for Gitpod Dedicated. We use the `gitpod-role` when installing this second template.
```
To invert the passive which should make it friendlier to readers

Author: We try to avoid using active / we in the docs because it creates a We vs. You dynamic. Hence why you are seeing this here. But I agree this sentence can be written better.

Author: ```suggestion
2. `gitpod-instance` Template: This template installs the infrastructure for Gitpod Dedicated. The role created by the `gitpod-role` template is used to execute this second template.
```
